### ISSUE TITLE
Should not be able to remove  a group if land is deeded to it.

### ISSUE THREAD

**Issue by nbtafelberg** [OpenSim-NGC/OpenSim-Sasquatch#60]
One of my residents created a group, then deeded some land to the group.

Then they deleted the group - but the land is still deeded to it....so we end up with weirdness on the land console.

So if you then do LAND CLEAR on the console to clear it, re-deed the land to a new group, if you restart the region it then goes back to the invalid group.

Weird.

**Comment by renevega** [comment]
Yes, there are a number of group related bugs. Please also file this one with core devs in their Opensimulator.org mantis as it would be best if they got to it first.
